src/components/content-block/content-block.scss

```diff
@@ -57,11 +58,14 @@ $content-block: null !default;
 
 .ecl-content-block__title {
   color: map.get($content-block, 'text-color');
-  font: map.get($content-block, 'title-font');
-  font-weight: map.get($content-block, 'title-font-weight');
   margin: map.get($content-block, 'title-margin');
   max-width: var(--max-w);
 
+  @include mixins.responsive-font(
+    map.get($content-block, 'title-font'),
+    map.get($theme, 'font-weight', 'regular')
```

Comment:
the font weight in the mixin is optional. If you are using regular, then there is no need to define it, you can just pass the font and that's it

Reply:
it might be in the next version, currently for the headings you get 300 by default